lib/customercontacttypes.php

```diff
@@ -112,9 +112,22 @@ function validate_customer_emails(&$customerdata, &$contacts, &$error)
     foreach ($customerdata['emails'] as $idx => &$val) {
         $email = trim($val['contact']);
         $name = trim($val['name']);
+        $properties = !empty($val['properties']) ? $val['properties'] : array();
         $type = !empty($val['type']) ? array_sum($val['type']) : null;
         $type |= CONTACT_EMAIL;
 
+        if (!empty($properties)) {
+            foreach ($properties as $pkey => $property) {
+                $propertyName = Utils::removeInsecureHtml($property['name']);
+                $propertyValue = Utils::removeInsecureHtml($property['value']);
```

Comment:
Nie jest to zbyt drastyczne? Głośno myślę ;-)

Reply:
Moim zdaniem łatwiej, w razie potrzeby, będzie to później rozluźnić niż wzmacniać kontrolę. Na razie na potrzeby 'email-subject' myślę, że to będzie dobre. Zobaczymy jakie jeszcze w przyszłości będą właściwości to najwyżej poluzujemy.